nextflow_schema.json

```diff
@@ -1418,6 +1418,34 @@
             },
             "fa_icon": "fas fa-angle-double-right"
         },
+
+        "bgc_gecco_convert": {
+            "title": "GECCO BGC conversion option",
+            "type": "object",
+            "description": "Settings for converting GECCO output files.",
+            "default": "",
+            "properties": {
+                "run_gecco_convert": {
+                    "type": "boolean",
+                    "default": false,
+                    "description": "Enable GECCO file conversion.",
+                    "help_text": "Converts GECCO output into formats like GFF3, GenBank, or FASTA for further analysis. Modifies tool parameter - gecco convert: `--run_gecco_convert true`"
+                },
+                "gecco_convert_mode": {
+                    "type": "string",
+                    "default": "clusters",
+                    "description": "Select conversion mode.",
+                    "help_text": "Either clusters or gbk folder output, depending on what is reformatted. Modifies tool parameter: - gecco convert: `--gecco_convert_mode `"
+                },
+                "gecco_convert_format": {
+                    "type": "string",
+                    "default": "gff",
+                    "description": "Set output format.",
+                    "help_text": "Choose output format: 'gff', 'fasta', or 'genbank'. Modifies tool parameter: - gecco convert: `--gecco_convert_format`"
```

Comment:
My previous comment from the first review has not been addressed yet https://github.com/nf-core/funcscan/pull/506#discussion_r2514190125

If you need more guidance just ask on Slack :)